src/main/java/seedu/address/model/person/RsvpStatus.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Guest's RsvpStatus status in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidRsvpStatus(String)}
+ */
+public class RsvpStatus {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "RsvpStatus Status should only have one of three values. true or false.";
+
+    /**
+     * RSVP status can only be one of the following values.
+     */
+    public enum PossibleRsvpStatus {
+        YES, NO, MAYBE
```

Comment:
would it be better to have UNKNOWN instead of MAYBE for guests that have yet to RSVP? MAYBE seems  a bit ambiguous